src/features/Apiexplorer/RequestResponseRenderer/index.tsx

```diff
@@ -58,7 +66,9 @@ function RequestResponseRenderer<T extends TSocketEndpointNames>({
           Clear
         </Button>
       </div>
-      {!is_logged_in && toggle_modal ? (
+      {is_valid ? (
+        <ValidDialog setIsValid={setIsValid} setToggleModal={setToggleModal} />
```

Comment:
@sanjam-deriv please update the dialog design, add some paddings and stuff, it looks ugly :D